src/lib.rs

```diff
@@ -65,6 +65,44 @@ impl Build {
         }
     }
 
+    #[cfg(windows)]
+    fn check_env_var(&self, var_name: &str) -> Option<bool> {
+        env::var_os(var_name).and_then(|s| {
+            let tmp = s
+                .to_str()
+                .expect("The environment variable has invalid Unicode")
+                .to_lowercase();
+            match &tmp[..] {
```

Comment:
I don't think there's any need to go all out here, let's just check for either 0 or 1

Reply:
Well, there are there situation we need consider: force-enable, force-disable and auto-detecting. In this case, I think 0 or 1 is not suitable. I think 'yes/y', 'no/n', 'auto/a' will be better. (And leaving it unset means auto) (I'll remove those 'true'/'false', whatever)
 What do you think?

Reply:
I personally don't really want to maintain an env var which is so meticulously parsed, I think a simple 0/1 is all we need. This isn't really going to be heavily used enough to warrant so much to handle it.